### PR TITLE
Fixes type sizes for Article Stub and Four Up Teaser

### PR DESCRIPTION
**Pull request by designerneil**
take away unnecessary parent selector in json

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4432: Article Stub and Four Up Teaser fix typography (https://issues.ama-assn.org/browse/EWL-4432)

## Description

`naleck Nick Aleck added a comment - 3 days ago
The text should be in the small desktop style, per the typography doc (Viewing in Chrome)`

Fixes typography size
Fixes json file and removes extraneous parent 

## To Test
- [ ] http://localhost:3000/?p=organisms-four-up-teaser
- [ ] Observe font sizes being smaller ask requested
- [ ] http://localhost:3000/?p=pages-news
- [ ] Observe the margin around the 4 up teaser

## Visual Regressions

none

## Relevant Screenshots/GIFs

screen shot 2018-01-26 at 2 53 02 pm

screen shot 2018-01-26 at 2 52 00 pm

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/four-up-teaser.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="ama__four-up-teaser">
-  {% for item in fourUpTeaser.items[0:4] %}
-    {% set articleStub = item %}
+  {% for article in fourUpTeaser %}
+    {% set articleStub = article %}
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/four-up-teaser.twig]: These items may not always be articles on the D8 website. It is probably best to just keep it as item.

File: styleguide/source/assets/scss/03-organisms/_four-up-teaser.scss [hunk 2]
```diff
@@ -12,28 +14,29 @@
   }
 
   a {
-    @include font-size($p-font-sizes);
+    @extend %ama__type--small;
```
Comment (labbydev) [inline styleguide/source/assets/scss/03-organisms/_four-up-teaser.scss]: Does this mixin include sizing for the line height? It seems like the font got smaller but the line height didn't scale appropriately.

**Review by labbydev — CHANGES_REQUESTED** [review]
@designerneil it looks like the line-height didn't get adjusted as the font-size was adjusted. Can you take a look at reducing that?

File: styleguide/source/assets/scss/03-organisms/_four-up-teaser.scss (hunk 2 above)
Comment (designerneil) [reply styleguide/source/assets/scss/03-organisms/_four-up-teaser.scss]: all line heights are governed by the font-variables css which gets fed into the font_mixins

**Review by labbydev — APPROVED** [review]

**Pull request merged**